src/app/components/organizacion/organizacion-prestaciones.component.ts

```diff
@@ -0,0 +1,160 @@
+import { IOrganizacion } from './../../interfaces/IOrganizacion';
+import { OrganizacionService } from './../../services/organizacion.service';
+import { Component, OnInit, HostBinding } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+import { Auth } from '@andes/auth';
+import { Plex } from '@andes/plex';
+import { TipoPrestacionService } from '../../services/tipoPrestacion.service';
+import { ITipoPrestacion } from '../../interfaces/ITipoPrestacion';
+
+@Component({
+    selector: 'organizacion-prestaciones',
+    templateUrl: 'organizacion-prestaciones.html',
+})
+export class OrganizacionOfertaPrestacionalComponent implements OnInit {
+    @HostBinding('class.plex-layout') layout = true;  // Permite el uso de flex-box en el componente
+    disabledPanel = true;
+    agregando = false;
+    editando = false;
+    prestacion: ITipoPrestacion;
+    detalle: string;
+    public idOrganizacion: String;
+    public organizacion: IOrganizacion;
+    public ofertaPrestacionalSeleccionada: any;
+    public modelo: any = { prestacion: null, descripcion: null };
+
+
+    constructor(
+        public plex: Plex,
+        private auth: Auth,
+        private router: Router,
+        private route: ActivatedRoute,
+        private organizacionService: OrganizacionService,
+        private conceptosTurneablesService: TipoPrestacionService,
+    ) { }
+
+    ngOnInit() {
+        this.getOrganizacion();
+    }
+
+    getOrganizacion() {
+        this.route.params.subscribe(params => {
+            this.idOrganizacion = params['id'];
+            if (!this.auth.check('tm:organizacion:sectores')) {
+                this.router.navigate(['inicio']);
+            }
+            this.organizacionService.getById(this.idOrganizacion).subscribe(org => {
+                this.organizacion = org;
+            });
+        });
+    }
+
+    loadTipoPrestaciones(event) {
+        this.conceptosTurneablesService.get({ turneable: 1 }).subscribe((data) => {
+            let dataF = data.filter(x => {
+                return this.auth.check('turnos:planificarAgenda:prestacion:' + x.id);
+            });
+            event.callback(dataF);
+        });
```

Comment:
Este bloque de código podría simplificarse prescindiendo de las llaves y returns innecesarios:
`this.conceptosTurneablesService.get({ turneable: 1 }).subscribe(data =>
        event.callback(data.filter(x => this.auth.check(`\``turnos:planificarAgenda:prestacion:${x.id}`\``)))
    );`